server/data/prisonRegisterApiTypes.ts

```diff
@@ -1,3 +1,3 @@
 import { components } from '../@types/prison-register-api'
 
-export type Prison = Pick<components['schemas']['PrisonDto'], 'prisonId' | 'prisonName'>
+export type PrisonRegisterPrison = Pick<components['schemas']['PrisonDto'], 'prisonId' | 'prisonName'>
```

Comment:
Prison register and visit scheduler now both have a `PrisonDto` type so giving this a more explicit name - and we only need two properties from it.